### PR TITLE
Text corrections

### PR DESCRIPTION
**Pull request by p-smith**
Label and subtitle corrections as listed in #983.

### CODE REVIEW

File: data/en/interlude/campaign_start.sub [hunk 1]
```diff
@@ -3,12 +3,12 @@
 00:20.500-00:21.500 At your command, Sir.
 00:24.000-00:26.800 The commander of a sector has just retired,
 00:27.000-00:31.000 and I'm assigning you to go there and to secure the area.
-00:31.500-00:35.800 The planet Achilles has been in state of war for some time now,
+00:31.500-00:35.800 The planet Achilles has been a in state of war for some time now,
```
Comment (akarnokd) [inline data/en/interlude/campaign_start.sub]: "in a state"

File: data/es/interlude/campaign_start.sub [hunk 2]
```diff
@@ -3,12 +3,12 @@
 00:20.500-00:21.500 At your command, Sir.
 00:24.000-00:26.800 The commander of a sector has just retired,
 00:27.000-00:31.000 and I'm assigning you to go there and to secure the area.
-00:31.500-00:35.800 The planet Achilles has been in state of war for some time now,
+00:31.500-00:35.800 The planet Achilles has been a in state of war for some time now,
```
Comment (akarnokd) [inline data/es/interlude/campaign_start.sub]: "in a state"

Comment (p-smith) [reply data/es/interlude/campaign_start.sub]: Apparently I can't even copypaste.

**Comment by akarnokd** [comment]
Thanks for tackling these!

**Pull request merged**